lib/wicked_pdf/wicked_pdf_helper.rb

```diff
@@ -95,7 +95,7 @@ def set_protocol(source)
       protocol = WickedPdf.config[:default_protocol] || 'http'
       if source[0, 2] == '//'
         source = [protocol, ':', source].join
-      elsif !source[0, 7].include?('://')
```

Comment:
This seems counter intuitive, since ```source[0] == "h"```, ```source[7] == "/"``, and ```source[8] == "c"```, but ```source[0,y]``` is giving us a string range, specifically the first y characters  -- ```source[0,0] ==  ""```, ```source[0,1] == "h"```, and ```source[0,7] == "https:/"```